### PR TITLE
add addStrictTransportSecurityHeader metatype prop

### PR DESCRIPTION
**Pull request by wtlucy**
For #13515

### CODE REVIEW

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -176,4 +176,7 @@ stopAppStartUponListenerException.name=Stop application for unhandled listener e
...
 
 redirectToRelativeUrl.name=Send redirect response to a relative URL location
-redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
\ No newline at end of file
+redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
+
+addStrictTransportSecurityHeader.name=Add the HTTP Strict Transport Security header for HTTPS responses
+addStrictTransportSecurityHeader.desc=Enables the HTTP Strict Transport Security (HSTS) header for HTTPS responses and allows a value for that header to be set. For example: "max-age=31536000;includeSubDomains". HSTS can also be configured in the web.xml via the "com.ibm.ws.webcontainer.ADD_STS_HEADER_WEBAPP" context-param.
```
Comment (dmuelle) [inline dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties:182]: Enables the HTTP Strict Transport Security (HSTS) header for HTTPS responses and allows a value to be set for that header, for example: "max-age=31536000;includeSubDomains". HSTS can also be configured in the web.xml file by setting the "com.ibm.ws.webcontainer.ADD_STS_HEADER_WEBAPP" context parameter.

File: dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -176,4 +176,7 @@ stopAppStartUponListenerException.name=Stop application for unhandled listener e
 stopAppStartUponListenerException.desc=Some web applications depend on context listeners for setup before the web application starts. When this property is set to true, the application stops starting up when an unhandled exception is thrown from the context listeners. For servlet-5.0 and newer Servlet features, the default value is true. 
 
 redirectToRelativeUrl.name=Send redirect response to a relative URL location
-redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location. 
\ No newline at end of file
+redirectToRelativeUrl.desc=Send redirect response to a relative URL location without processing it. Set this property to true to send redirect response without converting the URL to an absolute location.
```
Comment (dmuelle) [inline dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties:179]: suggested edit:
Send a redirect response to a relative URL location without processing the URL. Set this property to true to send a redirect response without converting the URL to an absolute location. 

I wonder if the first sentence is necessary? Kinda says the same thing twice

**Review by dmuelle — CHANGES_REQUESTED** [review]

**Review by dmuelle — APPROVED** [review]

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
#build

**Comment by wtlucy** [comment]
#run-libby-bot

**Pull request merged**